File: PHP/card.php
<?php
include_once("payment.php");

class Card extends Payment{
    public $cardNumber;
    public $cardDate;
    public $cardCvv;

    public function __construct($id, $amount, $cardNumber, $cardDate, $cardCvv){
        parent::__construct($id, $amount);
        $this->$cardNumber = $cardNumber;
        $this->$cardDate = $cardDate;
        $this->$cardCvv = $cardCvv;
    }
}
?>

File: Python/card.py
from payment import Payment

class Card(Payment):
    card_number = int
    card_date = str
    card_cvv = int


    def __init__(self, id, amount, card_number, card_date, card_cvv):
        super().__init__(id, amount)
        self.card_number = card_number
        self.card_date = card_date
        self.card_cvv = card_cvv
